doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1196,6 +1196,125 @@ Abbreviations
    denoted expression is performed at definition time. Type-checking is
    done only at the time of use of the abbreviation.
 
+
+Numeral notations
+-----------------
+
+Starting with version 8.9, the following command allows the user to customize
```

Comment:
"Starting with version 8.9": shall not resist long to time.

Reply:
Thanks, fixed.  (Sorry, I missed this one on my earlier pass.)